doc/issues.md

```diff
@@ -23,7 +23,7 @@ Returns an array of issues.
 $issues = $client->api('issue')->find('KnpLabs', 'php-github-api', 'closed', 'bug');
 ```
 
-Returns an array of closed issues matching the "bug" term.
+Returns an array of closed issues matching the "bug" term. For more complex searches, use the [search api](/doc/search.md) which supports the advanced github search syntax.
```

Comment:
Did I add an absolute URL somewhere?

> On Aug 5, 2015, at 9:48 AM, Christophe Coevoet <notifications@github.com> wrote:
> 
> In doc/issues.md:
> 
> > @@ -23,7 +23,7 @@ Returns an array of issues.
> >  $issues = $client->api('issue')->find('KnpLabs', 'php-github-api', 'closed', 'bug');
> >  ```
> >  
> > -Returns an array of closed issues matching the "bug" term.
> > +Returns an array of closed issues matching the "bug" term. For more complex searches, use the [search api](/doc/search.md) which supports the advanced github search syntax.
> please use relative links. It will be more reliable if the way to diplay the doc changes
> 
> —
> Reply to this email directly or view it on GitHub.
>